scripts/validate-artifacts.ps1

```diff
@@ -64,7 +103,14 @@
     }
 } else {
     Write-Output `u{1F914}
-    throw "Unable to find Public IP address resouce. Please make sure that is was migrated and try again."
+    throw "Unable to find Public IP address resouce. Please create a Public IP resouce (Basic SKU, dynamic IP allocation) and try again."
```

Comment:
The script checks for a Public IP resource with 'Basic' SKU and 'Dynamic' allocation. Ensure these settings match your deployment requirements to avoid validation errors.